kitchen-tests/kitchen.yml

```diff
@@ -84,15 +84,15 @@ platforms:
     image: dokken/centos-7
     pid_one_command: /usr/lib/systemd/systemd
     intermediate_instructions:
-      - RUN yum -y install e2fsprogs
+      # - RUN yum -y install e2fsprogs
```

Comment:
I think doing something like this is the right call:

```
sed -ibak -e 's!#baseurl=http://mirror.centos.org/$contentdir!baseurl=http://vault.centos.org!g' CentOS-*repo
```

Reply:
thanks!